target/arc/translate.c

```diff
@@ -106,6 +106,36 @@ static void gen_gotoi_tb(DisasContext *ctx, int n, target_ulong dest)
     }
 }
 
+static void
+arc_cpu_release_llockscond_locks(ARCCPU *cpu) {
+    struct lpa_lf_entry *entry = cpu->env.arconnect.lpa_lf;
+    QemuMutex *locked_mutex = cpu->env.arconnect.locked_mutex;
+    if (locked_mutex != NULL && entry != NULL) {
+        qemu_mutex_unlock(locked_mutex);
+        entry->lpa_lf = 0;
+    }
+}
+
+void arc_cpu_record_sigsegv(CPUState *cs, vaddr addr,
```

Comment:
Maybe move the comment here, as the function description.

Reply:
The comment inside the function describes a single action of the handler (currently the only action, but this might change in the future), not the purpose of the handler itself, so a more detailed comment based on TCGCPUOps description was added in .h file